lib/core/test/unit/Cardano/Pool/MetadataSpec.hs

```diff
@@ -118,22 +118,34 @@ dataDir = $( TH.LitE . TH.StringL <$>
 testUrl :: Port -> String
 testUrl p = "http://localhost:" <> show p <> "/testnet-stake-pool-registry-master.zip"
 
-presentPids :: [PoolId]
-presentPids = map PoolId
-    [ "pk1afhcpw2tg7nr2m3wr4x8jaa4dv7d09gnv27kwfxpjyvukwxs8qdqwg85xp"
-    , "pk1z4vh8gva25w07x8574uujuveu8gz43fu6qfln3t4prcavrvcphjsk0pdqs"
+presentOwners :: [PoolId]
+presentOwners = map PoolId
+    [ "ed25519_pk1afhcpw2tg7nr2m3wr4x8jaa4dv7d09gnv27kwfxpjyvukwxs8qdqwg85xp"
+    , "ed25519_pk1z4vh8gva25w07x8574uujuveu8gz43fu6qfln3t4prcavrvcphjsk0pdqs"
     ]
 
 presentMetas :: [StakePoolMetadata]
 presentMetas =
-    [ StakePoolMetadata (unsafeFromText "FST") "https://12345"
-        "ed25519_pk15vz9yc5c3upgze8tg5kd7kkzxqgqfxk5a3kudp22hdg0l2za00sq2ufkk7"
-    , StakePoolMetadata (unsafeFromText "TICK") "https://12345"
-        "ed25519_pk15vz9yc5c3upgze8tg5kd7kkzxqgqfxk5a3kudp22hdg0l2za00sq2ufkk7"
+    [ StakePoolMetadata
+        { ticker = unsafeFromText "FST"
+        , homepage = "https://12345"
+        , owner = unsafeFromText "ed25519_pk1afhcpw2tg7nr2m3wr4x8jaa4dv7d09gnv27kwfxpjyvukwxs8qdqwg85xp"
+        , name = "First stake pool"
+        , description = Just "It's better than SND"
+        , pledgeAddress = "addr15vz9yc5c3upgze8tg5kd7kkzxqgqfxk5a3kudp22hdg0l2za00sq2ufkk7"
+        }
+    , StakePoolMetadata
+        { ticker = unsafeFromText "TICK"
+        , homepage = "https://12345"
+        , owner = unsafeFromText "ed25519_pk1z4vh8gva25w07x8574uujuveu8gz43fu6qfln3t4prcavrvcphjsk0pdqs"
+        , name = "Pooley Mc-Poolface"
```

Comment:
**[question]**

What do you think about testing some _weird_ character sets in the name and/or description? Japanese and Chinese Kanjis, Cyrilic, Emojis and all sort of non-latin characters ?

Reply:
I think this is a good idea and we should do this through the QuickCheck generators.
My approach is to "ground" the tests using a fairly simple golden example, which is an actual zip file produced with the registry validator tool*.
Then attempt to test all the corner cases with randomly generated data.

(* however we need to wait for an update of the registry tool before we can download a proper example zip - the current one in this PR is synthetic).